app/utils/constants.js

```diff
@@ -38,3 +38,13 @@ export const MARK_AS_CORRECT_METHOD = 'mrkascorrect';
 
 export const EOS_IS_NOT_INIT = 'EOS is not initialized.';
 export const SCATTER_IN_NOT_INSTALLED = 'Scatter is not installed.';
+
+export const HUBSPOT_URL =
+  'https://api.hsforms.com/submissions/v3/integration/submit';
+export const HUBSPOT_PORTAL_ID = '5258952';
+export const HUBSPOT_SEND_EMAIL_FORM_ID =
+  '05305f90-6546-4718-88e8-2d2c8e79c00c';
+export const HUBSPOT_SEND_MESSAGE_FORM_ID =
+  'b75c88d0-ecc7-49b5-a69d-18b666f1d1ea';
+export const HUBSPOT_PAGE_URI = 'www.example.com/page';
```

Comment:
Page URI and page name are not constants. That should be URL and page name on Peerania website where user submitted the form from.
For email submission it can be landing or faq page. On landing page there are several places where user could submit email. These variables should let us know how submission happened.